xbmc/speech/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+set(HEADERS RecognizerErrors.h)
```

Comment:
I would suggest this speech folder gets added to treedata

https://github.com/xbmc/xbmc/blob/master/cmake/treedata/common/subdirs.txt

cmake passes treedata to access all the CMakeLists.txt files. in this particular case, as its just a header, it "works", but if we do this wrong from the start, if there ends up being code compiled (ie cpp), someone will tear their hair out down the line because of it.

Reply:
It's there already.

Reply:
sorry, must have missed that, my bad